docs/content/v2024.2/yugabyte-platform/manage-deployments/remove-nodes.md

```diff
@@ -12,6 +12,55 @@ menu:
 type: docs
 ---
 
+## Automatic YB-Master failover
+
+{{<tags/feature/ea>}} To avoid under-replication, YugabyteDB Anywhere can automatically detect YB-Master servers that are not responding to the master leader, as well as master servers that are lagging WAL operations, and fail over to another available node.
+
+### Prerequisites
+
+- Automatic YB-Master failover is {{<tags/feature/ea>}}. To enable the feature for a universe, set the **Auto Master Failover** Universe Runtime Configuration option (config key `yb.auto_master_failover.enabled`) to true. Refer to [Manage runtime configuration settings](../../administer-yugabyte-platform/manage-runtime-config/).
+- Universe running v2.20.3.0 or 2.21.0.0 or later, on-premises or on a cloud provider (Kubernetes is not supported).
+- The provider configuration for the universe includes a replacement node in the same availability zone.
+
+### Failover
+
+Automatic master failover works as follows:
+
+1. When active, by default YugabyteDB Anywhere checks universe masters every minute to see if they respond.
+
+    You can customize this interval using the universe runtime configuration option `yb.auto_master_failover.detect_interval`.
+
+1. YugabyteDB Anywhere declares a master is failed or potentially failed when one of the following conditions are met:
```

Comment:
when any one of the following conditions are met:

Both happening can also trigger.